csrankings-b.csv

```diff
@@ -436,6 +436,7 @@ Bharat Kumar Bhargava,Purdue University,https://www.cs.purdue.edu/people/bb,rhLs
 Bharat L. Bhuva,Vanderbilt University,http://engineering.vanderbilt.edu/bio/bharat-bhuva,NOSCHOLARPAGE
 Bharath Hariharan,Cornell University,http://home.bharathh.info,TpglobcAAAAJ
 Bhargab B. Bhattacharya,IIT Kharagpur,https://www.isical.ac.in/~bhargab,udxFQnwAAAAJ
+Bhargav Narayanan,Rutgers University,https://sites.math.rutgers.edu/~narayanan/,LRN9H2sAAAAJ

```

Comment:
No indication of how they meet inclusion criteria.

Reply:
They publish in CS venues https://dblp.org/pid/205/2840.html

Reply:
"They publish in CS venues" does not speak in any way to the inclusion criteria; only the affiliation counts, so this is fine.